tests/benchmarks/test_join.py

```diff
@@ -24,17 +19,21 @@ def test_join_big(small_client, mem_mult, shuffle):
     with dask.config.set(shuffle=shuffle):
         memory = cluster_memory(small_client)  # 76.66 GiB
 
-        df1_big = timeseries_of_size(memory * mem_mult)
-        df1_big["x2"] = df1_big["x"] * 1e9
-        df1_big = df1_big.astype({"x2": "int"})
+        df1_big = timeseries_of_size(
+            memory * mem_mult, dtypes={str(i): float for i in range(100)}
+        )  # 66.58 MiB partitions
+        df1_big["predicate"] = df1_big["0"] * 1e9
+        df1_big = df1_big.astype({"predicate": "int"})
 
-        df2_big = timeseries_of_size(memory * mem_mult)
+        df2_big = timeseries_of_size(
+            memory * mem_mult, dtypes={str(i): float for i in range(100)}
+        )  # 66.58 MiB partitions
 
         # Control cardinality on column to join - this produces cardinality ~ to len(df)
-        df2_big["x2"] = df2_big["x"] * 1e9
-        df2_big = df2_big.astype({"x2": "int"})
+        df2_big["predicate"] = df2_big["0"] * 1e9
```

Comment:
Out of curiosity, why did you choose the name "predicate"?

Reply:
That's me coming from a DB background. I wanted a name that's more descriptive in this context than `x2` and since it's the column used in the _join predicate_ (i.e., the expression used to merge the tables/dataframes), that's what I ended up with.

Reply:
I see. That's good, no need to change it, it was more out of curiosity and I just learn something new :) 
I'm merging this in.